native_client/BUILD

```diff
@@ -65,10 +67,16 @@ tf_cc_shared_object(
             "kiss_fft130/_kiss_fft_guts.h",
             "kiss_fft130/tools/kiss_fftr.h",
             "ds_version.h"] +
-           DECODER_SOURCES,
+            select({
+                "//tensorflow:windows": DECODER_SOURCESWIN,
+                "//conditions:default": DECODER_SOURCES,
+            }),
     # -Wno-sign-compare to silent a lot of warnings from tensorflow itself,
     # which makes it harder to see our own warnings
-    copts = ["-Wno-sign-compare", "-fvisibility=hidden"],
+    copts = select({ 
+        "//tensorflow:windows": [],
```

Comment:
Are we sure we don't need any flag? At least for parity with `-fvisibility=hidden` (but I don't know the flag on windows)

Reply:
We dont need these flags, -wno-sign-compare is the equivalent of /w for windows build tools, I will add it, but it still show warnings. 
And for -fvisibility it is related to the exports in [here](https://github.com/carlfm01/DeepSpeech/blob/934b965016f206518840206cff3a7308e016ad06/native_client/deepspeech.h#L8)
we use extern "C" for Windows, so we don't need -fvisibility flag

Reply:
Are you 100% sure? We had to do that to make sure embedding code would not end up weird calling wrong symbols. It's an edge case I had experimenting OpenCL on Linux, some protobuf symbols from our code would get picked up by some  xorg / Mesa / OpenCL lib that also used protobuf but with a different version.

Reply:
Yes, MSVC does not export any declaration unless you explicitly tell like [here](https://github.com/carlfm01/DeepSpeech/blob/934b965016f206518840206cff3a7308e016ad06/native_client/deepspeech.h#L5)
For GCC is the opposite, you have to explicitly hidde members.

https://stackoverflow.com/questions/12717991/symbol-visibility-in-windows

Reply:
removed the /w, further in the compilation was causing warnings with the /w overridden with W3